src/styles/components/_calendar.scss

```diff
@@ -0,0 +1,50 @@
+@import "../base/base";
+@import "../abstracts/variables";
+
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  width: $width;
+  gap: $gap;
+  padding: $padding;
+
+  @for $i from 1 through 31 {
+    &__day-#{$i} {
+      background-color: $cell-color;
+      width: $cell-dimensions;
+      height: $cell-dimensions;
+      box-sizing: border-box;
+      border: 1px solid black;
+      display: flex;
+      justify-content: center;
+      align-items: center;
+      cursor: pointer;
+    }
+    &__day-#{$i}:hover {
+      background-color: pink;
+      transform: $cell-animation;
+      animation-duration: 0.5s;
+    }
+  }
+
+  @for $i from 1 through 31 {
+    &__day-#{$i}::after {
+      content: "#{$i}";
+      font-size: $font;
+    }
+  }
+
+  // dodanie margin-left w zależności od wybranego dnia tygodnia
```

Comment:
It is a good thing that you are adding comments for more complex parts of code. But please keep them in English.